<launch>
  <node name="main_cam" pkg="usb_cam" type="usb_cam_node" output="log" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="yuyv" />
    <param name="io_method" value="mmap"/>
    <param name="framerate" value="15" />
    <param name="autofocus" value="true" />
  </node>
</launch>
